spec/routes/web/private_subnet_spec.rb

```diff
@@ -76,7 +76,7 @@
         click_button "Create"
 
         expect(page.title).to eq("Ubicloud - #{name}")
-        expect(page).to have_flash_notice("'#{name}' will be ready in a few seconds")
+        expect(page).to have_content "'#{name}' will be ready in a few seconds"
```

Comment:
This seems unrelated.  Possible incorrect merge/rebase (same for other change in this file)?